### ISSUE TITLE
Add 'inbounds' display range to `geom_football`

### ISSUE THREAD

**Issue by nguyenank** [sportsdataverse/sportyR#18]
Add a default display range for football geoms that only displays what's in play.

**Comment by rossdrucker** [comment]
I've gone ahead and tried to implement this for all surfaces, not just football. The football-specific result looks like this:

```r
geom_football(
  "nfl",
  "in bounds only"
  color_updates = list(
    "sideline" = "#ff552e",
    "end_line" = "#13294b"
  )
)
```

![Example of in bounds only display range](https://user-images.githubusercontent.com/15791518/195303230-371e2ec5-9c8f-4a97-b9d2-ce3c4181ce52.png)

_Note: Colors of sideline and end line changed to make them stand out from background of plot_

Is that what you had in mind?

**Comment by nguyenank** [comment]
Yep; that's pretty much exactly what I was thinking!

**Comment by rossdrucker** [comment]
Closed via #19
